internal/controller/imagepolicy_controller.go

```diff
@@ -371,7 +361,7 @@ func (r *ImagePolicyReconciler) reconcile(ctx context.Context, sp *patch.SerialP
 		// If there's no tag in the database, mark not ready and retry.
 		if err == errNoTagsInDatabase {
 			conditions.MarkFalse(obj, meta.ReadyCondition, imagev1.DependencyNotReadyReason, "%s", err)
-			result, retErr = ctrl.Result{}, err
+			result, retErr = ctrl.Result{}, nil
```

Comment:
We no longer retry, please update the comment above

Reply:
Fixed, thanks!